modules/apps/27-interchain-accounts/controller/keeper/keeper.go

```diff
@@ -61,6 +62,15 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s-%s", host.ModuleName, icatypes.ModuleName))
 }
 
+// GetConnectionID returns the connection id for the given port and channelIDs.
+func (k Keeper) GetConnectionID(ctx sdk.Context, portID, channelID string) (string, error) {
```

Comment:
I chose to add this function rather than export the channelKeeper, as so far it's the only time we've needed.